### PR TITLE
Updated subarray devices and TANGO version

### PR DESCRIPTION
**Pull request by njthykkathu**
Updated SDPSubarray - It creates 2 servers and 2 devices
Updated TANGO version to TANGO 9.3.3rc
New Images for:
SDP Master - nexus.engageska-portugal.pt/sdp-prototype/tangods_sdp_master:0.2.3
SDP Subarray - nexus.engageska-portugal.pt/sdp-prototype/tangods_sdp_subarray:0.5.16

### CODE REVIEW

File: charts/sdp-prototype/templates/sdp-devices-deploy.yml [hunk 1]
```diff
@@ -74,20 +74,39 @@ spec:
         env:
         - name: TANGO_HOST
           value: databaseds-tango-base-{{ .Release.Name }}:10000
-        image: nexus.engageska-portugal.pt/sdp-prototype/tangods_sdp_master:0.2.1
+        image: nexus.engageska-portugal.pt/sdp-prototype/tangods_sdp_master:0.2.3
         imagePullPolicy: Always
```
Comment (Matteo04052017) [inline charts/sdp-prototype/templates/sdp-devices-deploy.yml]: If you put the named version, what's the need for Always as pull policy

File: charts/sdp-prototype/templates/sdp-devices-deploy.yml [hunk 2]
```diff
@@ -74,20 +74,39 @@ spec:
...
+        - name: TOGGLE_CBF_OUTPUT_LINK
+          value: "0"
+        - name: TOGGLE_AUTO_REGISTER
+          value: "0"
         - name: SDP_CONFIG_HOST
           value: {{ include "sdp-prototype.etcd-host" . }}
-        image: nexus.engageska-portugal.pt/sdp-prototype/tangods_sdp_subarray:0.5.12
+        args: ["2", "-v4"]
+        image: nexus.engageska-portugal.pt/sdp-prototype/tangods_sdp_subarray:0.5.16
         imagePullPolicy: Always
```
Comment (Matteo04052017) [inline charts/sdp-prototype/templates/sdp-devices-deploy.yml]: as above

File: charts/sdp-prototype/data/sdp-devices.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
     "servers": {
-        "SdpMaster": {
+        "SDPMaster": {
```
Comment (Matteo04052017) [inline charts/sdp-prototype/data/sdp-devices.json]: Is there going to be any problem with the devices that references this one? i.e. Does the TM know about this name change?

File: charts/sdp-prototype/templates/sdp-devices-deploy.yml (hunk 1 above)
Comment (njthykkathu) [reply charts/sdp-prototype/templates/sdp-devices-deploy.yml]: These new images have been tested by the NCRA team and have confirmed that it is working. 
I am not sure what other ways we could test it.

File: charts/sdp-prototype/data/sdp-devices.json (hunk 3 above)
Comment (njthykkathu) [reply charts/sdp-prototype/data/sdp-devices.json]: There shouldn't be a problem caused by these changes.

File: charts/sdp-prototype/templates/sdp-devices-deploy.yml (hunk 1 above)
Comment (Matteo04052017) [reply charts/sdp-prototype/templates/sdp-devices-deploy.yml]: I'll talk with them. Never mind. Thanks.

Comment (njthykkathu) [reply charts/sdp-prototype/templates/sdp-devices-deploy.yml]: sure will do.

**Pull request closed without merging**